docs/tutorials/fundamentals/viewer.md

```diff
@@ -259,6 +259,7 @@ viewer.layers.pop(i)
 One of the main strengths of **napari** is that it has been designed from the beginning to handle n-dimensional data. While much consumer photography is 2D and `RGB`, scientific image data can often be volumetric (i.e. 3D), volumetric timeseries (i.e. 4D), or even higher dimensional. **napari** places no limits on the dimensionality of its input data for all its layer types.
 
 Adding data with a dimensionality greater than 2D will cause dimension sliders to appear directly underneath the canvas and above the status bar. As many sliders as needed will appear to ensure the data can be fully browsed. For example, a 3D dataset needs one slider, a 4D dataset needs two sliders, and so on. The widths of the scroll bars of the dimension sliders are directly related to how many slices are in each dimension.
+To the left of each slider will be an integer indicating which dimension is being controlled by that slider. These integers are automatically updated when changing which dimensions are to be displayed.
```

Comment:
Strictly speaking, there isn't always an int, because the axis slider could have a label?
Not sure if it adds confusion, but maybe it's worth adding mention of:
`viewer.dims.order` and `viewer.dims.axis_labels`
A bit of scope creep, but it fits this section of the tutorial and those arn't documented outside the API as far as I can tell.

Reply:
Will adjust when I get back home, but then we really need a method for changing 'current_step'.